fxmanifest.lua

```diff
@@ -1,21 +1,32 @@
 -- For support join my discord: https://discord.gg/Z9Mxu72zZ6
 
 author "Andyyy#7666"
-description "Clothing store for ND framework"
-version "2.0.0"
+description "Clothing store for ND framework with blocked items support"
+version "2.1.0"
 
 fx_version "cerulean"
 game "gta5"
 lua54 "yes"
 
+-- Shared scripts
 shared_scripts {
     "@ND_Core/init.lua",
     "@ox_lib/init.lua",
     "config.lua"
 }
-server_script "source/server.lua"
-client_script "source/client.lua"
 
+-- Server scripts
+server_scripts {
+    "source/server.lua"
+}
+
+-- Client scripts
+client_scripts {
+    "source/blockedItems.lua",  -- Added blocked items script

```

Comment:
You don't need to add blockedItems into client_scripts you can simply add it to files and use lib.load function from ox_lib. See here [ND_Characters/fxmanifest.lua#L20](https://github.com/ND-Framework/ND_Characters/blob/main/fxmanifest.lua#L20) and here [ND_Characters/source/client.lua#L12](https://github.com/ND-Framework/ND_Characters/blob/main/source/client.lua#L12) as example.